src/Bindings/CMakeLists.txt

```diff
@@ -168,7 +168,7 @@ set_source_files_properties(${BINDING_OUTPUTS} PROPERTIES GENERATED TRUE)
 set_source_files_properties(${CMAKE_SOURCE_DIR}/src/Bindings/Bindings.cpp PROPERTIES COMPILE_FLAGS -Wno-error)
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-	set_source_files_properties(Bindings.cpp PROPERTIES COMPILE_FLAGS ${COMPILE_FLAGS} "-Wno-old-style-cast -Wno-missing-prototypes")
+	set_source_files_properties(Bindings.cpp PROPERTIES COMPILE_FLAGS "-Wno-old-style-cast -Wno-missing-prototypes -Wno-zero-as-null-pointer-constant")
```

Comment:
Because it is irritating. You may think, that it resolves to `-Wno-error`. But actually it resolves to nothing :trollface: , because there is no variable named `COMPILE_FLAGS`. It is only a property, which you can store in a variable via [get_target_property](https://cmake.org/cmake/help/latest/command/get_target_property.html).